app/scripts/modules/kubernetes/v2/serverGroup/details.controller.ts

```diff
@@ -0,0 +1,71 @@
+import { IController, module } from 'angular';
+
+import {
+  Application,
+  CONFIRMATION_MODAL_SERVICE,
+  IServerGroup,
+  SERVER_GROUP_READER,
+  SERVER_GROUP_WARNING_MESSAGE_SERVICE,
+  SERVER_GROUP_WRITER,
+  ServerGroupReader
+} from '@spinnaker/core';
+
+import { IKubernetesServerGroup } from './IKubernetesServerGroup';
+
+interface IServerGroupFromStateParams {
+  accountId: string;
+  region: string;
+  name: string;
+}
+
+class KubernetesServerGroupDetailsController implements IController {
+  public state = { loading: true };
+  public serverGroup: IKubernetesServerGroup;
+
+  constructor(serverGroup: IServerGroupFromStateParams,
+              public app: Application,
+              private serverGroupReader: ServerGroupReader) {
+    'ngInject';
+
+    this.app
+      .ready()
+      .then(() => this.extractServerGroup(serverGroup))
+      .catch(() => this.autoClose());
+  }
+
+  public canResizeServerGroup(): boolean {
+    return false; // will change with daemon set
+  }
+
+  private autoClose(): void {
+    return;
+  }
+
+  private transformServerGroup(serverGroupDetails: IServerGroup): IKubernetesServerGroup {
+    const serverGroup: IKubernetesServerGroup = Object.assign(serverGroupDetails);
+    const [apiVersion, kind, name] = serverGroup.name.split('|');
+    serverGroup.displayName = name;
+    serverGroup.kind = kind;
+    serverGroup.apiVersion = apiVersion.replace('.', '/');
```

Comment:
Not sure what form `apiVersion` takes, but JS `.replace` will only swap the first match. You'll need `.replace(/\./g, '/')` if you want all of them.

Reply:
I don't think there are any w/ more than one slash, but who know what api versions will get added